lib/kamal/configuration/proxy.rb

```diff
@@ -1,8 +1,13 @@
 class Kamal::Configuration::Proxy
   include Kamal::Configuration::Validation
 
+  HTTP_PORT = 80
+  HTTPS_PORT = 443
+  LOG_MAX_SIZE = "10m"
+  MINIMUM_VERSION = "v0.8.2"
```

Comment:
Nice clean up!

I was also looking at proxy configuration options, hoping too override the version with a newer one - see #1252